content/posts/03_system_desing_interview/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "How do you conduct and evaluate a system design interview?"
+date: 2023-02-26
+tags: [
+    "interview",
+]
+draft: true,
+---
+
+1. Intro
+  * different approaches, based on the company requirements
+  * helps to evaluate and distinguish senior and staff+ candidates
+  * difficult to do well
+2. Goals of the sys design interview
+  * High level goals the same as for all interviews
+  * Would the candidate be able to do their job
```

Comment:
I guess this fits very nicely into `High level goals the same as for all interviews`.

Reply:
Yes, but it's just outline